src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -420,6 +422,45 @@ class AccountControllerTest @Autowired constructor(
         }
     }
 
+    @NestedTest
+    @DisplayName("POST /accounts/changePassword/{id}")
+    inner class ChangePassword {
+        private val password = "test_password"
+        private val changePasswordAccount: Account = ObjectMapper().readValue(
+            ObjectMapper().writeValueAsString(testAccount),
+            Account::class.java
+        )
+
+        @BeforeEach
+        fun addAccount() {
+            changePasswordAccount.password = encoder.encode(changePasswordAccount.password)
```

Comment:
Can this line be in a before all or init-like method? Only needs to be done once (and would actually fail if there were more test methods inside this inner class)